home/models.py

```diff
@@ -1466,6 +1454,26 @@ class SubmissionStatus(models.TextChoices):
         use_json_field=True,
     )
 
+    submission_status = models.CharField(
+        max_length=30,
+        choices=SubmissionStatus.choices,
+        blank=True,
+        default=SubmissionStatus.NOT_SUBMITTED_YET,
+    )
+
+    submission_result = models.TextField(
+        help_text="The result of submitting the template",
+        null=True,
```

Comment:
I know this was here before, but we probably don't want this to be nullable as well as blank, since it's a string type: https://docs.djangoproject.com/en/5.0/ref/models/fields/#null